monai/fl/client/monai_algo.py

```diff
@@ -325,31 +295,25 @@ def _add_config_files(self, config_files):
 class MonaiAlgo(ClientAlgo, MonaiAlgoStats):
     """
     Implementation of ``ClientAlgo`` to allow federated learning with MONAI bundle configurations.
-    FIXME: reimplement this class based on the bundle "ConfigWorkflow".
 
     Args:
-        bundle_root: path of bundle.
+        bundle_root: directory path of the bundle.
+        train_workflow: the bundle workflow to execute training.
+        eval_workflow: the bundle workflow to execute evaluation.
         local_epochs: number of local epochs to execute during each round of local training; defaults to 1.
         send_weight_diff: whether to send weight differences rather than full weights; defaults to `True`.
-        config_train_filename: bundle training config path relative to bundle_root. Can be a list of files;
-            defaults to "configs/train.json".
-        config_evaluate_filename: bundle evaluation config path relative to bundle_root. Can be a list of files.
-            If "default", config_evaluate_filename = ["configs/train.json", "configs/evaluate.json"] will be used;
+        config_train_filename: bundle training config path relative to bundle_root. can be a list of files.
+            defaults to "configs/train.json". only useful when `train_workflow` is None.
+        config_evaluate_filename: bundle evaluation config path relative to bundle_root. can be a list of files.
+            if "default", ["configs/train.json", "configs/evaluate.json"] will be used.
+            this arg is only useful when `eval_workflow` is None.
         config_filters_filename: filter configuration file. Can be a list of files; defaults to `None`.
-        disable_ckpt_loading: do not use any CheckpointLoader if defined in train/evaluate configs; defaults to `True`.
```

Comment:
Let me think about it whether we can disable checkpoint loader but avoid modifying config directly, because we suppose to support both config-based and python-based bundles.

Thanks.

Reply:
Hi @holgerroth ,

I added the "disable checkpoint" logic back, now it only works for config-based bundles, for python-based or any other customized bundles, it's hard to guess the handlers info, suppose users to disable it manually.
I also updated the test `evaluate` config to cover it, Could you please help take a look again?

Thanks in advance.